worker/src/worker.js

```diff
@@ -0,0 +1,29 @@
+const throng = require('throng');
+const dotenv = require('dotenv');
+const Queue = require('bull');
+const logger = require('./logger');
+const { processFile } = require('./files');
+
+dotenv.config(process.cwd(), '../.env');
+
+// TODO: Change these to support cloud.gov as well as local dev
+const REDIS_HOST = process.env.REDIS_HOST || 'redis://127.0.0.1';
+const REDIS_PORT = process.env.REDIS_PORT || '6379';
+const { REDIS_PASS } = process.env;
+
+// Number of workers to spawn
+const workers = process.env.WEB_CONCURRENCY || 2;
+
+// Number of jobs per worker. Can be adjusted if clamav is getting bogged down
+const maxJobsPerWorker = process.env.MAX_JOBS_PER_WORKER || 5;
+
+// Pull jobs off the redis queue and process them.
+function start() {
+  const scanQueue = new Queue('scan', `redis://${REDIS_HOST}:${REDIS_PORT}`, { redis: { password: REDIS_PASS } });
+  scanQueue.on('failed', (job, error) => logger.error(`job ${job.data.key} failed with error ${error}`));
```

Comment:
This line should use the auditLogger I mentioned before, but the on completed logging should stay the same as it is here.